src/server.c

```diff
@@ -5630,7 +5631,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "lru_clock:%u\r\n", server.lruclock,
             "executable:%s\r\n", server.executable ? server.executable : "",
             "config_file:%s\r\n", server.configfile ? server.configfile : "",
-            "io_threads_active:%i\r\n", server.io_threads_active));
```

Comment:
we should keep this line for compat.

```suggestion
            "io_threads_active:%i\r\n", server.io_threads_active_num > 1));
```